// react-pure-render-utils/index.js
'use strict';

exports.__esModule = true;
exports.pureMethod = exports.pureClass = exports.pureStateless = exports.PureMixin = exports.PureComponent = exports.shouldPureComponentUpdate = exports.shallowEqual = undefined;

var _decorators = require('./decorators');

Object.defineProperty(exports, 'pureClass', {
  enumerable: true,
  get: function get() {
    return _decorators.pureClass;
  }
});
Object.defineProperty(exports, 'pureMethod', {
  enumerable: true,
  get: function get() {
    return _decorators.pureMethod;
  }
});

var _shallowEqual2 = require('./shallowEqual');

var _shallowEqual3 = _interopRequireDefault(_shallowEqual2);

var _function = require('./function');

var _function2 = _interopRequireDefault(_function);

var _component = require('./component');

var _component2 = _interopRequireDefault(_component);

var _mixin = require('./mixin');

var _mixin2 = _interopRequireDefault(_mixin);

var _highOrder = require('./high-order');

var _highOrder2 = _interopRequireDefault(_highOrder);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.shallowEqual = _shallowEqual3.default;
exports.shouldPureComponentUpdate = _function2.default;
exports.PureComponent = _component2.default;
exports.PureMixin = _mixin2.default;
exports.pureStateless = _highOrder2.default;
